shared/src/test/scala/FutureTimeoutTests.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2016-2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import scala.concurrent._
+import scala.concurrent.duration._
+import org.scalatest._
+import cats.data.NonEmptyList
+import scala.language.postfixOps
```

Comment:
You shouldn't need this as we have `"-language:postfixOps"` as a `scalacOption`.